packages/jest-diff/src/diffStrings.js

```diff
@@ -24,6 +24,14 @@ export type DiffOptions = {|
 
 type Diff = {diff: string, isDifferent: boolean};
 
+type Hunk = {|
```

Comment:
Yup, as stated in docs, structuredPatch returns an array of hunks (can't link it properly through phone) https://www.npmjs.com/package/diff